npm-shrinkwrap.json

```diff
@@ -10,7 +10,7 @@
       "integrity": "sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==",
```

Comment:
I removed that file `npm-shrinkwrap.json` from the contribution because we usually do not change it with command PR, but separately.

Reply:
thx need to write that down, second time I missed this. Sorry